server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/single_page_apps/agents.js.msx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+requirejs([
+  'jquery', 'mithril', 'lodash',
+  'helpers/form_helper', 'helpers/mithril_component_mixins',
+  'js-routes',
+  'models/agents/agents',
+  'models/agents/resources',
+  'models/agents/environments',
+  'views/agents/agents_widget',
+  'views/agents/agent_spa',
+  'foundation.util.mediaQuery',
+  'foundation.dropdownMenu',
+  'foundation.responsiveToggle',
+  'foundation.dropdown'
+], function ($, m, _,
+             f, ComponentMixins,
+             JsRoutes,
+             Agents, Resources, Environments, AgentsWidget, AgentsSPA) {
```

Comment:
Most of these components are unused `_`, `f`, `ComponentMixins`, `Agents`, `AgentsWidget`, isn't the IDE telling you about these?